src/System.Private.Xml/tests/System.Private.Xml.Tests.builds

```diff
@@ -3,65 +3,125 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="Readers\CharCheckingReader\System.Xml.RW.CharCheckingReader.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
       <OSGroup>Windows_NT</OSGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
+    <Project Include="Readers\CharCheckingReader\System.Xml.RW.CharCheckingReader.Tests.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+    </Project>
     <Project Include="Readers\CustomReader\System.Xml.RW.CustomReader.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
That's correct, as discussed in https://github.com/dotnet/corefx/issues/12483 as well. I'm going to add that to this PR soon.